src/components/Modals/AssetSearch/AssetSearchModal.tsx

```diff
@@ -10,35 +10,27 @@ import {
 import type { Asset } from '@shapeshiftoss/asset-service'
 import { useCallback } from 'react'
 import { useTranslate } from 'react-polyglot'
+import type { AssetSearchProps } from 'components/AssetSearch/AssetSearch'
 import { AssetSearch } from 'components/AssetSearch/AssetSearch'
 import { useModal } from 'hooks/useModal/useModal'
 import { useWindowSize } from 'hooks/useWindowSize/useWindowSize'
 import { breakpoints } from 'theme/theme'
 
-type AssetSearchModalProps = {
-  onClick: (asset: Asset) => void
-  filterBy?: (assets: Asset[]) => Asset[] | undefined
-  disableUnsupported?: boolean
-}
-
-export const AssetSearchModal = ({
-  onClick,
-  filterBy,
-  disableUnsupported,
-}: AssetSearchModalProps) => {
+export const AssetSearchModal = ({ onClick, filterBy, disableUnsupported }: AssetSearchProps) => {
```

Comment:
While diffing this, might as well
```suggestion
export const AssetSearchModal: React.FC<AssetSearchProps> = ({ onClick, filterBy, disableUnsupported }) => {
```